modules/plugins/languages/nix.nix

```diff
@@ -175,6 +175,12 @@ in {
 
   config = mkIf cfg.enable (mkMerge [
     {
+      assertions = [
+        {
+          assertion = cfg.format.type != "nixpkgs-fmt";
+          message = "nixpkgs-fmt has been deprecated.";
```

Comment:
Expand this: what to do instead?

Reply:
Added a list of valid formatters to use instead as well as clarify `nixpkgs-fmt`'s situation.